src/main/webapp/script.js

```diff
@@ -12,19 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-google.charts.load('current', {'packages':['corechart']});
-google.charts.setOnLoadCallback(getChart);
-
 const CHART_WIDTH = 800;
 const CHART_HEIGHT = 400;
+const HIGHEST_SCORE = 1.0;
+const LOWEST_SCORE = -1.0;
+const INTERVAL = 0.2;
+
+google.charts.load('current', {'packages':['corechart']});
+google.setOnLoadCallback(getChart)
 
 function getYouTubeComments() { 
   const urlInput = document.getElementById('url-entry');
```

Comment:
Currently this method returns a Promise.

Is this intended? If so maybe this should be renamed to `getYoutubeCommentsPromise()` or `getYoutubeCommentsAsync()`.